[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building entity 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:3.1.0:java (default-cli) @ entity ---
当前时间戳（秒）: 1747557932
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/public/home/blockchain_2/slave2/.m2/repository/org/slf4j/slf4j-reload4j/1.7.36/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/public/home/blockchain_2/slave2/.m2/repository/org/apache/logging/log4j/log4j-slf4j-impl/2.17.1/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
log4j:WARN No appenders could be found for logger (org.apache.commons.beanutils.converters.BooleanConverter).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747557935
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747557935
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747557935
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747557935
org.janusgraph.core.JanusGraphException: Could not execute operation due to backend exception
	at org.janusgraph.diskstorage.util.BackendOperation.execute(BackendOperation.java:54)
	at org.janusgraph.diskstorage.BackendTransaction.executeRead(BackendTransaction.java:521)
	at org.janusgraph.diskstorage.BackendTransaction.edgeStoreMultiQuery(BackendTransaction.java:300)
	at org.janusgraph.graphdb.database.StandardJanusGraph.edgeMultiQuery(StandardJanusGraph.java:568)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx.lambda$executeMultiQuery$4(StandardJanusGraphTx.java:1252)
	at org.janusgraph.graphdb.query.profile.QueryProfiler.profile(QueryProfiler.java:113)
	at org.janusgraph.graphdb.query.profile.QueryProfiler.profile(QueryProfiler.java:105)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx.executeMultiQuery(StandardJanusGraphTx.java:1252)
	at org.janusgraph.graphdb.query.vertex.MultiVertexCentricQueryBuilder.execute(MultiVertexCentricQueryBuilder.java:121)
	at org.janusgraph.graphdb.query.vertex.MultiVertexCentricQueryBuilder.edges(MultiVertexCentricQueryBuilder.java:172)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.VertexStepBatchFetcher.makeQueryAndExecute(VertexStepBatchFetcher.java:39)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.MultiQueriableStepBatchFetcher.prefetchNextBatch(MultiQueriableStepBatchFetcher.java:112)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.MultiQueriableStepBatchFetcher.fetchData(MultiQueriableStepBatchFetcher.java:96)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.JanusGraphVertexStep.flatMap(JanusGraphVertexStep.java:117)
	at org.apache.tinkerpop.gremlin.process.traversal.step.map.FlatMapStep.processNextStart(FlatMapStep.java:49)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.next(AbstractStep.java:140)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.next(AbstractStep.java:40)
	at org.apache.tinkerpop.gremlin.process.traversal.Traversal.fill(Traversal.java:184)
	at org.apache.tinkerpop.gremlin.process.traversal.Traversal.toList(Traversal.java:122)
	at entity.SG1_dec_base.processInitAddress(SG1_dec_base.java:111)
	at entity.SG1_dec_base.main(SG1_dec_base.java:37)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:279)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.janusgraph.diskstorage.TemporaryBackendException: Could not successfully complete backend operation due to repeated temporary exceptions after PT8M20S
	at org.janusgraph.diskstorage.util.BackendOperation.executeDirect(BackendOperation.java:98)
	at org.janusgraph.diskstorage.util.BackendOperation.execute(BackendOperation.java:52)
	... 22 more
Caused by: org.janusgraph.diskstorage.TemporaryBackendException: Temporary failure in storage backend
	at org.janusgraph.diskstorage.hbase.HBaseKeyColumnValueStore.getHelper(HBaseKeyColumnValueStore.java:230)
	at org.janusgraph.diskstorage.hbase.HBaseKeyColumnValueStore.getSlice(HBaseKeyColumnValueStore.java:119)
	at org.janusgraph.diskstorage.keycolumnvalue.KCVSProxy.getSlice(KCVSProxy.java:87)
	at org.janusgraph.diskstorage.keycolumnvalue.cache.ExpirationKCVSCache.getSlice(ExpirationKCVSCache.java:122)
	at org.janusgraph.diskstorage.BackendTransaction$2.call(BackendTransaction.java:303)
	at org.janusgraph.diskstorage.BackendTransaction$2.call(BackendTransaction.java:300)
	at org.janusgraph.diskstorage.util.BackendOperation.executeDirect(BackendOperation.java:66)
	... 23 more
Caused by: org.apache.hadoop.hbase.client.RetriesExhaustedException: Failed after attempts=16, exceptions:
2025-05-18T08:46:35.768Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=93,methodName=Get], waitTime=60021ms, rpcTimeout=60000ms
2025-05-18T08:47:35.899Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=109,methodName=Get], waitTime=60008ms, rpcTimeout=60000ms
2025-05-18T08:48:36.120Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=125,methodName=Get], waitTime=60011ms, rpcTimeout=60000ms
2025-05-18T08:49:36.440Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=141,methodName=Get], waitTime=60009ms, rpcTimeout=60000ms
2025-05-18T08:50:36.960Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=157,methodName=Get], waitTime=60009ms, rpcTimeout=60000ms
2025-05-18T08:51:37.980Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=173,methodName=Get], waitTime=60004ms, rpcTimeout=60000ms
2025-05-18T08:52:40.015Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=189,methodName=Get], waitTime=60001ms, rpcTimeout=60000ms
2025-05-18T08:53:44.061Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=206,methodName=Get], waitTime=60008ms, rpcTimeout=60000ms
2025-05-18T08:54:54.091Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=224,methodName=Get], waitTime=60008ms, rpcTimeout=60000ms
2025-05-18T08:56:04.181Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=242,methodName=Get], waitTime=60007ms, rpcTimeout=60000ms
2025-05-18T08:57:14.270Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=260,methodName=Get], waitTime=60008ms, rpcTimeout=60000ms
2025-05-18T08:58:24.370Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=278,methodName=Get], waitTime=60005ms, rpcTimeout=60000ms
2025-05-18T08:59:44.440Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=298,methodName=Get], waitTime=60007ms, rpcTimeout=60000ms
2025-05-18T09:01:04.471Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=318,methodName=Get], waitTime=60004ms, rpcTimeout=60000ms
2025-05-18T09:02:24.660Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=338,methodName=Get], waitTime=60002ms, rpcTimeout=60000ms
2025-05-18T09:03:44.741Z, RpcRetryingCaller{globalStartTime=2025-05-18T08:45:35.683Z, pause=100, maxAttempts=16}, org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=358,methodName=Get], waitTime=60006ms, rpcTimeout=60000ms

	at org.apache.hadoop.hbase.client.RpcRetryingCallerImpl.callWithRetries(RpcRetryingCallerImpl.java:141)
	at org.apache.hadoop.hbase.client.HTable.get(HTable.java:390)
	at org.apache.hadoop.hbase.client.HTable.lambda$get$0(HTable.java:363)
	at org.apache.hadoop.hbase.trace.TraceUtil.trace(TraceUtil.java:216)
	at org.apache.hadoop.hbase.client.HTable.get(HTable.java:363)
	at org.apache.hadoop.hbase.client.HTable.lambda$get$1(HTable.java:409)
	at org.apache.hadoop.hbase.trace.TraceUtil.trace(TraceUtil.java:216)
	at org.apache.hadoop.hbase.client.HTable.get(HTable.java:407)
	at org.janusgraph.diskstorage.hbase.HBaseKeyColumnValueStore.getHelper(HBaseKeyColumnValueStore.java:198)
	... 29 more
Caused by: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node10:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=358,methodName=Get], waitTime=60006ms, rpcTimeout=60000ms
	at org.apache.hadoop.hbase.ipc.IPCUtil.wrapException(IPCUtil.java:222)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient.onCallFinished(AbstractRpcClient.java:392)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient.access$100(AbstractRpcClient.java:92)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient$3.run(AbstractRpcClient.java:426)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient$3.run(AbstractRpcClient.java:421)
	at org.apache.hadoop.hbase.ipc.Call.setTimeout(Call.java:107)
	at org.apache.hadoop.hbase.ipc.RpcConnection$1.run(RpcConnection.java:134)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at org.apache.hbase.thirdparty.io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	... 1 more
Caused by: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=358,methodName=Get], waitTime=60006ms, rpcTimeout=60000ms
	at org.apache.hadoop.hbase.ipc.RpcConnection$1.run(RpcConnection.java:135)
	... 6 more
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 1 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
org.apache.tinkerpop.gremlin.process.traversal.util.TraversalInterruptedException
	at org.janusgraph.diskstorage.BackendTransaction.executeRead(BackendTransaction.java:525)
	at org.janusgraph.diskstorage.BackendTransaction.edgeStoreMultiQuery(BackendTransaction.java:300)
	at org.janusgraph.graphdb.database.StandardJanusGraph.edgeMultiQuery(StandardJanusGraph.java:568)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx.lambda$executeMultiQuery$4(StandardJanusGraphTx.java:1252)
	at org.janusgraph.graphdb.query.profile.QueryProfiler.profile(QueryProfiler.java:113)
	at org.janusgraph.graphdb.query.profile.QueryProfiler.profile(QueryProfiler.java:105)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx.executeMultiQuery(StandardJanusGraphTx.java:1252)
	at org.janusgraph.graphdb.query.vertex.MultiVertexCentricQueryBuilder.execute(MultiVertexCentricQueryBuilder.java:121)
	at org.janusgraph.graphdb.query.vertex.MultiVertexCentricQueryBuilder.edges(MultiVertexCentricQueryBuilder.java:172)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.VertexStepBatchFetcher.makeQueryAndExecute(VertexStepBatchFetcher.java:39)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.MultiQueriableStepBatchFetcher.prefetchNextBatch(MultiQueriableStepBatchFetcher.java:112)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.MultiQueriableStepBatchFetcher.fetchData(MultiQueriableStepBatchFetcher.java:96)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.JanusGraphVertexStep.flatMap(JanusGraphVertexStep.java:117)
	at org.apache.tinkerpop.gremlin.process.traversal.step.map.FlatMapStep.processNextStart(FlatMapStep.java:49)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.next(AbstractStep.java:140)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.next(AbstractStep.java:40)
	at org.apache.tinkerpop.gremlin.process.traversal.Traversal.fill(Traversal.java:184)
	at org.apache.tinkerpop.gremlin.process.traversal.Traversal.toList(Traversal.java:122)
	at entity.SG1_dec_base.processInitAddress(SG1_dec_base.java:116)
	at entity.SG1_dec_base.main(SG1_dec_base.java:46)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:279)
	at java.lang.Thread.run(Thread.java:748)
Over Processing Init Address


发现 0 个匹配组：
java.net.SocketTimeoutException: callTimeout=60000, callDuration=60121: Call to address=node11:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=678,methodName=Scan], waitTime=60016ms, rpcTimeout=60000ms row 'bnb_deanonymization,\x80\x00\x00\x05\x81\xE9\xA3\x80,99999999999999' on table 'hbase:meta' at region=hbase:meta,,1.1588230740, hostname=node11,16020,1747110088632, seqNum=-1
	at org.apache.hadoop.hbase.client.RpcRetryingCallerImpl.callWithRetries(RpcRetryingCallerImpl.java:157)
	at org.apache.hadoop.hbase.client.ResultBoundedCompletionService$QueueingFuture.run(ResultBoundedCompletionService.java:74)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call to address=node11:16020 failed on local exception: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=678,methodName=Scan], waitTime=60016ms, rpcTimeout=60000ms
	at org.apache.hadoop.hbase.ipc.IPCUtil.wrapException(IPCUtil.java:222)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient.onCallFinished(AbstractRpcClient.java:392)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient.access$100(AbstractRpcClient.java:92)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient$3.run(AbstractRpcClient.java:426)
	at org.apache.hadoop.hbase.ipc.AbstractRpcClient$3.run(AbstractRpcClient.java:421)
	at org.apache.hadoop.hbase.ipc.Call.setTimeout(Call.java:107)
	at org.apache.hadoop.hbase.ipc.RpcConnection$1.run(RpcConnection.java:134)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at org.apache.hbase.thirdparty.io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at org.apache.hbase.thirdparty.io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	... 1 more
Caused by: org.apache.hadoop.hbase.ipc.CallTimeoutException: Call[id=678,methodName=Scan], waitTime=60016ms, rpcTimeout=60000ms
	at org.apache.hadoop.hbase.ipc.RpcConnection$1.run(RpcConnection.java:135)
	... 6 more
Over Processing Init Address


发现 3 个匹配组：
org.apache.tinkerpop.gremlin.process.traversal.util.TraversalInterruptedException
	at org.janusgraph.diskstorage.BackendTransaction.executeRead(BackendTransaction.java:525)
	at org.janusgraph.diskstorage.BackendTransaction.edgeStoreMultiQuery(BackendTransaction.java:300)
	at org.janusgraph.graphdb.database.StandardJanusGraph.edgeMultiQuery(StandardJanusGraph.java:568)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx.lambda$executeMultiQuery$4(StandardJanusGraphTx.java:1252)
	at org.janusgraph.graphdb.query.profile.QueryProfiler.profile(QueryProfiler.java:113)
	at org.janusgraph.graphdb.query.profile.QueryProfiler.profile(QueryProfiler.java:105)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx.executeMultiQuery(StandardJanusGraphTx.java:1252)
	at org.janusgraph.graphdb.query.vertex.MultiVertexCentricQueryBuilder.execute(MultiVertexCentricQueryBuilder.java:121)
	at org.janusgraph.graphdb.query.vertex.MultiVertexCentricQueryBuilder.edges(MultiVertexCentricQueryBuilder.java:172)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.VertexStepBatchFetcher.makeQueryAndExecute(VertexStepBatchFetcher.java:39)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.MultiQueriableStepBatchFetcher.prefetchNextBatch(MultiQueriableStepBatchFetcher.java:112)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.fetcher.MultiQueriableStepBatchFetcher.fetchData(MultiQueriableStepBatchFetcher.java:96)
	at org.janusgraph.graphdb.tinkerpop.optimize.step.JanusGraphVertexStep.flatMap(JanusGraphVertexStep.java:117)
	at org.apache.tinkerpop.gremlin.process.traversal.step.map.FlatMapStep.processNextStart(FlatMapStep.java:49)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.next(AbstractStep.java:140)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.next(AbstractStep.java:40)
	at org.apache.tinkerpop.gremlin.process.traversal.Traversal.fill(Traversal.java:184)
	at org.apache.tinkerpop.gremlin.process.traversal.Traversal.toList(Traversal.java:122)
	at entity.SG1_dec_base.processInitAddress(SG1_dec_base.java:116)
	at entity.SG1_dec_base.main(SG1_dec_base.java:46)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:279)
	at java.lang.Thread.run(Thread.java:748)
Over Processing Init Address


发现 1 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747561680
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747561680
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747561680
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747561681
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747561681
Over Processing Init Address


发现 0 个匹配组：
Over Processing Init Address


发现 0 个匹配组：
当前时间戳（秒）: 1747561791
[WARNING] thread Thread[Idle-Rpc-Conn-Sweeper-pool-0,5,entity.SG1_dec_base] was interrupted but is still alive after waiting at least 14997msecs
[WARNING] thread Thread[Idle-Rpc-Conn-Sweeper-pool-0,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-1,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RpcClient-timer-pool-0,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-2,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-3,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-4,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-5,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-6,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-7,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-8,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[ForkJoinPool.commonPool-worker-11,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-9,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] thread Thread[RPCClient-NioEventLoopGroup-1-10,5,entity.SG1_dec_base] will linger despite being asked to die via interruption
[WARNING] NOTE: 13 thread(s) did not finish despite being asked to via interruption. This is not a problem with exec:java, it is a problem with the running code. Although not serious, it should be remedied.
[WARNING] Couldn't destroy threadgroup org.codehaus.mojo.exec.ExecJavaMojo$IsolatedThreadGroup[name=entity.SG1_dec_base,maxpri=10]
java.lang.IllegalThreadStateException
	at java.lang.ThreadGroup.destroy(ThreadGroup.java:778)
	at org.codehaus.mojo.exec.ExecJavaMojo.execute(ExecJavaMojo.java:319)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:04 h
[INFO] Finished at: 2025-05-18T17:50:06+08:00
[INFO] Final Memory: 148M/17742M
[INFO] ------------------------------------------------------------------------
